SRC/NBASIC/nbas111e.txt

```diff
@@ -165,7 +165,7 @@ function description).
 only if you will not use the compiler.
 
 - Segment 2 is always connected to page 2 (addreses &H8000 to &HBFFF), and 
-contains the BASIC program being executed and maybe some variables.
+contains the BASIC program being executed and part of its' variables.

```

Comment:
The `its'` here should be just `its`, right?

Reply:
Yes, that's right.